doc/source/whatsnew/v0.23.0.txt

```diff
@@ -571,6 +585,7 @@ Other API Changes
 - Set operations (union, difference...) on :class:`IntervalIndex` with incompatible index types will now raise a ``TypeError`` rather than a ``ValueError`` (:issue:`19329`)
 - :class:`DateOffset` objects render more simply, e.g. "<DateOffset: days=1>" instead of "<DateOffset: kwds={'days': 1}>" (:issue:`19403`)
 - :func:`pandas.merge` provides a more informative error message when trying to merge on timezone-aware and timezone-naive columns (:issue:`15800`)
+- :func:`Timedelta.__mod__`, :func:`Timedelta.__divmod__` now accept timedelta-like and numeric arguments instead of raising ``TypeError`` (:issue:`19365`)
```

Comment:
This is the same as above? Isn't it enough to mention it once?

Reply:
This is an attempt to follow the instructions in #19365, some of which were unclear.  I'm open to suggestions.

Reply:
My suggestion would be to just remove this one, but I don't know the reason for the previous instruction

Reply:
@jrback is there consensus on this suggestion?

Reply:
oh i guess I wasn't clear. if you make a sub-section then no reason to repeat things, so kill this entry as it dupes the sub-section.